tests/installation/ipxe_install.pm

```diff
@@ -103,7 +103,8 @@ sub set_bootscript {
             $cmdline_extra .= " ssh=1 sshpassword=$testapi::password ";    # trigger ssh-text installation
         }
         else {
-            $cmdline_extra .= " vnc=1 VNCPassword=$testapi::password ";    # trigger default VNC installation
+            # trigger default graphical installation via SSH or VNC
+            $cmdline_extra .= " ssh=1 sshpassword=$testapi::password vnc=1 VNCPassword=$testapi::password ";
```

Comment:
This would be cleaner:

    $cmdline_extra .= " ssh=1 sshpassword=$testapi::password ";
    $cmdline_extra .= " vnc=1 VNCPassword=$testapi::password " unless check_var('VIDEOMODE', 'text');

Reply:
you are very right